src/xnatdcat/xnat_parser.py

```diff
@@ -67,18 +67,24 @@ def xnat_to_DCATDataset(project: XNATSession, config: Dict) -> DCATDataSet:
 
     creator_vcard = [
         VCard(
-            full_name=Literal(f"{project.pi.title or ''} {project.pi.firstname} {project.pi.lastname}".strip()),
+            full_name=Literal(f"{project.pi.title +' ' or ''}{project.pi.firstname} {project.pi.lastname}".strip()),
```

Comment:
a minor point but why string concatenation there if you strip spaces anyway?

Reply:
Good one, removed as I had forgotten about the strip (even tho it's right there)